### PR TITLE
Adds ML Watch Examples

### PR DESCRIPTION
**Pull request by blaklaybul**
This PR adds an example watch configuration for use with ML along with an additional configuration that utilizes the email functionality.

### CODE REVIEW

**Comment by blaklaybul** [comment]
Rich has pointed out that there are already existing watcher examples in [Alerting](https://github.com/elastic/examples/tree/master/Alerting/Sample%20Watches/ml_examples). I think we should merge this with the existing work, and add Rich's more creative examples here. 

1. https://gist.github.com/richcollier/7e5603c366b9fcece6f1a8b1b3cf4d3f
2. https://gist.github.com/richcollier/1c2b8161286bdca6c553859f28d3d66d
3. https://gist.github.com/richcollier/13e245c1c8c8cfc38ecb12e324ce301e

File: Alerting/Sample Watches/ml_examples/default_ml_watch.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+      }
+    },
+    "condition": {
+      "compare": {
+        "ctx.payload.aggregations.bucket_results.doc_count": {
+          "gt": 0
+        }
+      }
+    },
+    "actions": {
```
Comment (sophiec20) [inline Alerting/Sample Watches/ml_examples/default_ml_watch.json:212]: I think there was an email action for the default Watch. Can you please dbl check and include if it is available. This included the link back to the anomaly explorer which is useful.

**Comment by blaklaybul** [comment]
added the email example. this is ready for review.

**Comment by richcollier** [comment]
LGTM

**Pull request merged**